### PR TITLE
Add `extensionFromMime` utility.

### PR DESCRIPTION
**Pull request by kenzieschmoll**
This also bumps the version to 0.9.7 in preparation for release.

### CODE REVIEW

File: lib/src/mime_type.dart [hunk 1]
```diff
@@ -25,6 +25,19 @@ int get defaultMagicNumbersMaxLength => _globalResolver.magicNumbersMaxLength;
 String lookupMimeType(String path, {List<int> headerBytes}) =>
     _globalResolver.lookup(path, headerBytes: headerBytes);
 
+/// Returns the extension for the given MIME type.
+///
+/// If there are multiple extensions for [mime], return the first occurrence in
+/// the map. If there are no extensions for [mime], return [mime].
+String extensionFromMime(String mime) {
+  for (final key in defaultExtensionMap.keys) {
```
Comment (kevmoo) [inline lib/src/mime_type.dart:33]: may considering doing `in defaultExtensionMaps.entries` – it's a tiny allocation, but then you avoid hitting the content of the map for every key 🤷

File: lib/src/mime_type.dart [hunk 2]
```diff
@@ -25,6 +25,19 @@ int get defaultMagicNumbersMaxLength => _globalResolver.magicNumbersMaxLength;
...
+    if (defaultExtensionMap[key].toLowerCase() == mime.toLowerCase()) {
```
Comment (kevmoo) [inline lib/src/mime_type.dart:34]: consider doing `mime = mime.toLowerCase()` at the top of the function – convert once.

Also consider just adding a test that validates all of the values in the map `value == value.toLowerCase` then you can avoid doing the conversion for each value

File: lib/src/mime_type.dart (hunk 1 above)
Comment (kenzieschmoll) [reply lib/src/mime_type.dart]: done

File: lib/src/mime_type.dart (hunk 2 above)
Comment (kenzieschmoll) [reply lib/src/mime_type.dart]: done

**Pull request merged**